stock_checkout_sync/wizards/stock_move_checkout_sync.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class StockMoveCheckoutSync(models.TransientModel):
+    _name = "stock.move.checkout.sync"
+    _description = "Stock Move Checkout Sync"
+
+    picking_ids = fields.Many2many(comodel_name="stock.picking", required=True)
+    dest_picking_id = fields.Many2one(
+        comodel_name="stock.picking",
+        required=True,
+        readonly=True,
+        help="Destination operation for these moves.",
+    )
+    # in case we have several steps to sync all the moves,
+    # this field will contain them
+    remaining_help = fields.Html(readonly=True)
+    done_dest_picking_ids = fields.Many2many(
+        comodel_name="stock.picking",
+        relation="stock_move_checkout_sync_stock_picking_done_dest_rel",
+    )
+    show_skip_button = fields.Boolean(default=False)
```

Comment:
i guess this should be enough
```suggestion
    show_skip_button = fields.Boolean()
```